packages/schemas/src/lib/auth/auth-schemas.ts

```diff
@@ -1,19 +1,19 @@
-import { AUTH_METHOD_TYPE_VALUES } from '@lit-protocol/constants';
+import {
+  AUTH_METHOD_TYPE,
+  AUTH_METHOD_TYPE_VALUES,
+} from '@lit-protocol/constants';
 import { z } from 'zod';
 import {
   AuthMethodSchema,
   HexPrefixedSchema,
   NodeSetSchema,
   SessionKeyUriSchema,
 } from '../schemas';
+import { ScopeStringSchema } from './ScopeSchema';
```

Comment:
```suggestion
import { ScopeSchemaRaw } from './ScopeSchema';
```
properly transform scope strings like "sign-anything" to their BigInt values (like 1n).